packages/bbui/src/InlineAlert/InlineAlert.svelte

```diff
@@ -49,6 +51,20 @@
       >
     </div>
   {/if}
+  {#if link && linkText}
+    <div id="docs-link">
+      <a href={link} target="_blank" rel="noopener noreferrer" class="doc-link">
+        {linkText}
+        <svg
+          class="spectrum-Icon spectrum-Icon--sizeS spectrum-Toast-typeIcon"
+          focusable="false"
+          aria-hidden="true"
+        >
+          <use xlink:href="#spectrum-icon-18-LinkOut" />
+        </svg>
```

Comment:
It would be nicer to use our Icon component (just import it from "../Icon/Icon.svelte"), but it'll function the same! Style wise I would just make it a touch smaller and with a little space between the link and the text.

Reply:
Using our icon, you can make it smaller by just doing
```svelte
<Icon name="LinkOut" size="XS" /> 
```
for example.